test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/AuthenticationTests.cs

```diff
@@ -306,12 +306,8 @@ public class AuthenticationTests
             }
         }
 
-        [ConditionalTheory]
-        [InlineData(AuthenticationSchemes.Negotiate)]
-        [InlineData(AuthenticationSchemes.NTLM)]
-        // [InlineData(AuthenticationSchemes.Digest)]
-        [InlineData(AuthenticationSchemes.Basic)]
-        public async Task AuthTypes_Forbid_Forbidden(AuthenticationSchemes authType)
+        [ConditionalFact]
```

Comment:
cc @Tratcher can you look at this change especially? The theory parameter was never used in the test.